Android.bp

```diff
@@ -0,0 +1,2265 @@
+package {
+    default_applicable_licenses: ["external_intel_media_driver_license"],
+}
+
+license {
+    name: "external_intel_media_driver_license",
+    visibility: [":__subpackages__"],
+    license_text: [
+        "LICENSE.md",
+    ],
+}
+
+//####################COMMON CFLAGS######################
+common_cflags = [
+    "-Wreorder",
+    "-Wsign-promo",
+    "-Wno-invalid-offsetof",
+    "-fno-use-cxa-atexit",
+    "-frtti",
+    "-fexceptions",
+    "-fcheck-new",
+    "-Werror",
+    "-Wno-unknown-pragmas",
+    "-Wno-comments",
+    "-Wno-attributes",
+    "-Wno-narrowing",
+    "-Wno-overflow",
+    "-Wno-parentheses",
+    "-Wno-delete-incomplete",
+    "-Wno-overloaded-virtual",
+    "-std=c++14",
+    "-m64",
+    "-DNO_RTTI",
+    "-DNO_EXCEPTION_HANDLING",
+    "-Wno-ignored-optimization-argument",
+    "-finline-limit=100",
+    "-Wno-ignored-qualifiers",
+    "-Wno-missing-braces",
+    "-Wno-implicit-fallthrough",
+    "-DGMM_LIB_DLL",
+    "-DCLASS_TRACE=0",
+    "-DENABLE_KERNELS",
+    "-DENABLE_XE_KMD",
+    "-DHEVC_FEI_ENABLE_CMRT",
+    "-DIGFX_ARL_SUPPORTED",
+    "-DIGFX_DG2_SUPPORTED",
+    "-DIGFX_GEN11_ICLLP_SUPPORTED",
+    "-DIGFX_GEN11_JSL_SUPPORTED",
+    "-DIGFX_GEN11_SUPPORTED",
+    "-DIGFX_GEN12_ADLN_SUPPORTED",
+    "-DIGFX_GEN12_ADLP_SUPPORTED",
+    "-DIGFX_GEN12_ADLS_SUPPORTED",
+    "-DIGFX_GEN12_DG1_SUPPORTED",
+    "-DIGFX_GEN12_RKL_SUPPORTED",
+    "-DIGFX_GEN12_SUPPORTED",
+    "-DIGFX_GEN12_TGLLP_CMFCPATCH_SUPPORTED",
+    "-DIGFX_GEN12_TGLLP_CMFC_SUPPORTED",
+    "-DIGFX_GEN12_TGLLP_SUPPORTED",
+    "-DIGFX_GEN12_TGLLP_SWSB_SUPPORTED",
+    "-DIGFX_GEN8_BDW_SUPPORTED",
+    "-DIGFX_GEN8_SUPPORTED",
+    "-DIGFX_GEN9_BXT_SUPPORTED",
+    "-DIGFX_GEN9_CFL_SUPPORTED",
+    "-DIGFX_GEN9_CML_SUPPORTED",
+    "-DIGFX_GEN9_CMPV_SUPPORTED",
+    "-DIGFX_GEN9_GLK_SUPPORTED",
+    "-DIGFX_GEN9_KBL_SUPPORTED",
+    "-DIGFX_GEN9_SKL_SUPPORTED",
+    "-DIGFX_GEN9_SUPPORTED",
+    "-DIGFX_LNL_SUPPORTED",
+    "-DIGFX_MTL_SUPPORTED",
+    "-DIGFX_XE2_HPG_CMFCPATCH_SUPPORTED",
+    "-DIGFX_XE2_HPG_SUPPORTED",
+    "-DIGFX_XEHP_SDV_CMFCPATCH_SUPPORTED",
+    "-DIGFX_XEHP_SDV_SUPPORTED",
+    "-DIGFX_XE_HPG_CMFCPATCH_SUPPORTED",
+    "-DIGFX_XE_HPG_SUPPORTED",
+    "-DMEDIA_VERSION=\"24.3.4\"",
```

Comment:
Android.bp can be generated now by running python3 Tools/MediaDriverTools/Android/GenerateAndroidBp.py. Pre-requisite is libva and gmmlib are compiled and installed in the build machine.

When GenerateAndroidBp.py script is used, Version and version_details gets generated instead of hardcoding.